addons/hw_drivers/iot_handlers/drivers/PrinterDriver_W.py

```diff
@@ -55,6 +55,12 @@ def __init__(self, identifier, device):
         })
 
         self.receipt_protocol = 'escpos'
+        if any(cmd in device['iddentifer'] for cmd in ['STAR', 'Receipt']):#Find the correct STR to find
+            self.device_subtype = "receipt_printer"
+        elif "ZPL" in device['identifier']:
+            self.device_subtype = "label_printer"
+        else:
+            self.device_subtype = "office_printer"
```

Comment:
```Python
self.device_subtype = "office_printer"
```
This doesn't correspond to the model's "other_printer" which itself corresponds to "Office Printer".